examples/00-fluent/frozen_rotor_workflow.py

```diff
@@ -0,0 +1,424 @@
+pump_head.output_parameter = True
+
+
+################################################################################################################
+# Define Report Definitions
+# ==============================================================================================================
+
+# Create a report definition
+# p-out
+solver.settings.solution.report_definitions.surface.create("p-out")
+
+outlet_pressure_report_def = solver.settings.solution.report_definitions.surface[
+    "p-out"
+]
+outlet_pressure_report_def.report_type = "surface-massavg"
+outlet_pressure_report_def.surface_names = ["mass-flow-inlet-11"]
+outlet_pressure_report_def.field = "total-pressure"
+outlet_pressure_report_def.per_surface = False
+
+outlet_pressure_report_plot = solver.settings.solution.monitor.report_plots.create(
+    "p-out-rplot"
+)
+outlet_pressure_report_plot.report_defs = "p-out"
+
+outlet_pressure_report_file = solver.settings.solution.monitor.report_files.create(
+    "p-out-rfile"
+)
+outlet_pressure_report_file.report_defs = "p-out"
+
+# p-in
+solver.settings.solution.report_definitions.surface.create("p-in")
+inlet_pressure_report_def = solver.settings.solution.report_definitions.surface["p-in"]
+inlet_pressure_report_def.report_type = "surface-massavg"
+inlet_pressure_report_def.surface_names = ["inlet"]
+inlet_pressure_report_def.field = "total-pressure"
+inlet_pressure_report_def.per_surface = False
+
+
+# Pump Head
+pump_head_report_def = (
+    solver.settings.solution.report_definitions.single_valued_expression.create(
+        "pump-head"
+    )
+)
+pump_head_report_def = (
+    solver.settings.solution.report_definitions.single_valued_expression["pump-head"]
+)
+pump_head_report_def.definition = "head"
+
+# report plot
+pump_head_report_plot = solver.settings.solution.monitor.report_plots.create(
+    "pump-head-rplot"
+)
+pump_head_report_plot.report_defs = "pump-head"
+
+# report file
+pump_head_report_file = solver.settings.solution.monitor.report_files.create(
+    "pump-head-rfile"
+)
+pump_head_report_file.report_defs = "pump-head"
+
+# p-blade
+solver.settings.solution.report_definitions.surface.create("p-blade")
+blade_pressure_report_def = solver.settings.solution.report_definitions.surface[
+    "p-blade"
+]
+blade_pressure_report_def.report_type = "surface-massavg"
+blade_pressure_report_def.surface_names = ["blade"]
+blade_pressure_report_def.field = "pressure"
+blade_pressure_report_def.per_surface = False
+
+
+################################################################################################################
+# Initialization and run solver
+# ==============================================================================================================
+initialization = solver.settings.solution.initialization
+initialization.reference_frame = "absolute"
+initialization.hybrid_init_options.general_settings.initialization_options.initial_pressure = (
+    True
+)
+initialization.hybrid_initialize()
```

Comment:
@abhishekchitwar You can check, but I believe that `initialization.hybrid_initialize()` will be invoked automatically due to `run_calculation.calculate()`.